mteb/evaluation/evaluators/PairClassificationEvaluator.py

```diff
@@ -164,11 +166,7 @@ def _compute_metrics(
 
         return {
             "accuracy": float(acc),
-            "accuracy_threshold": float(acc_threshold),
             "f1": float(f1),
-            "f1_threshold": float(f1_threshold),
-            "precision": float(precision),
-            "recall": float(recall),
```

Comment:
Would prefer not to loose metrics

Reply:
I removed they because previously they weren't added to dataset